paket.dependencies

```diff
@@ -5,3 +5,4 @@ source https://www.nuget.org/api/v2
 nuget Autofac ~> 4.8
 nuget Moq ~> 4.10
 nuget Messerli.TempDirectory ~> 0.2
+nuget System.Collections.Immutable ~> 1.3
```

Comment:
Add a terminating newline

Reply:
Addressed in https://github.com/messerli-informatik-ag/test-utility/pull/11/commits/8a0c0e56a527bfa53be42bf21ad0598a499433ab